.github/workflows/ci.yml

```diff
@@ -91,3 +91,37 @@ jobs:
         with:
           command: test
           args: -p rspotify -p rspotify-http -p rspotify-model -p rspotify-macros --no-default-features --features=${{ matrix.features }}
+          
+  wasm-test:
+    name: Test for wasm build
+    runs-on: ubuntu-latest
+    env:
+      RSPOTIFY_CLIENT_ID : e1dce60f1e274e20861ce5d96142a4d3
+      RSPOTIFY_CLIENT_SECRET : 0e4e03b9be8d465d87fc32857a4b5aa3
```

Comment:
Is it necessary to use these environment variables to run the test? Moreover, it's better to store these values as [Action secrets](https://docs.github.com/en/actions/security-guides/encrypted-secrets) instead of defining in configuration file explicitly(it's dangerous to leak this information to the whole world)